.github/workflows/acceptance-tests.yml

```diff
@@ -69,31 +52,16 @@ jobs:
         fail-fast: true
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-    - name: Debugging
-      run: |
-        php --version
-        composer --version
-        docker --version
-        docker-compose --version
-
     - name: Install dependencies
-      run: |
-        if [ "${{ matrix.wp }}" == "nightly" ]
-        then
-          composer require --dev --update-with-dependencies --prefer-dist roots/wordpress-full="dev-main"
-        else
-          composer require --dev --update-with-dependencies --prefer-dist roots/wordpress-full="~${{ matrix.wp }}.0"
-        fi
+      env:
+        WP_CORE_CONSTRAINT: ${{ matrix.wp == 'nightly' && 'dev-main' || format('~{0}.0', matrix.wp) }}
+      uses: ramsey/composer-install@v2
```

Comment:
Why do I need to use a third party action for this?

Reply:
As this action does caching, `composer install` or `composer update`, handles timeout, output and more.
Viktor uses it in dozens of projects.

Reply:
You must know Ben!

Reply:
Alternatively you could have own workflows in _community health files_ https://github.com/ergebnis/.github/tree/main/actions/composer

Reply:
@johnbillion Would you merge something based on `plugin-infrastructure`?

Reply:
Not just yet, thanks Viktor. I'm making my way through a list of to-do items as I find the the time and based on the similar but not identical testing setup that I have across my plugins.

Reply:
I'd like to promote Ben's action.